cosmwasm/contracts/crosschain-registry/src/execute.rs

```diff
@@ -40,6 +41,82 @@ pub struct ContractAliasInput {
     pub new_alias: Option<String>,
 }
 
+pub fn propose_pfm(
+    ctx: (DepsMut, Env, MessageInfo),
+    chain: String,
+) -> Result<Response, ContractError> {
+    let (deps, env, info) = ctx;
+
+    let chain = chain.to_lowercase();
+
+    // Store the chain to validate. If validation fails this will be reverted
+    CHAIN_PFM_MAP.save(deps.storage, &chain, &ChainPFM::default())?;
+    // TODO: Check if the chain exists and only allow override on certain conditions
```

Comment:
Just noticed I forgot this todo. Who should be allowed to override this? I'd say anyone that can manage that chain.

This is different to the other registries because we're actually validating it, so overriding it would only unset it until the response is received.

Reply:
Two cases:
 * Chain had PFM, but then removes it
 * Someone tried to add PFM to the chain but it didn't validate (we would have the old attempt in storage). I think the right call here is to always replace it if it isn't approved so we can try to validate again

If we assume chains will not remove PFM this becomes simpler. Otherwise the question is still: who can change this?